src/utils/communitySbt/seasonsConfig.ts

```diff
@@ -0,0 +1,10 @@
+export const goerliSeasonLeavesCid: Record<number, string> = {
+    0: "QmYjnH1btsV5of8NHmjUUmK5va7kGBoVDTUwWy1HtLyq7d",
```

Comment:
goerliSeasonLeavesCid[0] is same as mainnetSeasonLeavesCid[0]... On purpose?

Reply:
on purpose -> there's a TODO below (after deployment on mainnet badges, replace CID)